server/apps/distributed-app/docker-compose-with-pulsar.yml

```diff
@@ -0,0 +1,83 @@
+version: '3'
+
+services:
+
+  james:
+    image: apache/james:distributed-latest
+    container_name: james
+    hostname: james.local
+    command:
+      - --generate-keystore
+    networks:
+      - james
+    ports:
+      - "80:80"
+      - "25:25"
+      - "110:110"
+      - "143:143"
+      - "465:465"
+      - "587:587"
+      - "993:993"
+      - "8000:8000"
+    volumes:
+      - ./sample-configuration/pulsar.properties:/root/conf/pulsar.properties
+      - ./sample-configuration/keystore:/root/conf/keystore
```

Comment:
You do not override the default `queue.properties` (with RabbitMQ) by a Pulsar mail queue choice?